drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -350,17 +349,29 @@ def do_attempt(self, data, suffix=''):
         self._validate_do_attempt()
 
         self.attempts += 1
-        self._mark_complete_and_publish_grade()  # must happen before _get_feedback
+        # pylint: disable=fixme
+        # TODO: Refactor this method to "freeze" item_state and pass it to methods that need access to it.
```

Comment:
@cahrens I've asked @bradenmacdonald about this and haven't got an answer yet. My current understanding is that it's not a good time to further increase the scope of v2.1 with optional refactroings, so it won't be a part of v2.1.